### ISSUE TITLE
PD: User testing branch w/ FF stuck on

### ISSUE THREAD

**Issue by IanLondon** [Opentrons/opentrons#7281]
Make a branch for Morgan to use with multi-select mode already toggled on, so that users don't have to toggle it themselves. This should not be merged into edge.

This is blocked by all the UI tweaks stuff

**Comment by IanLondon** [comment]
we made it! a few weeks ago
